modules/security/keyvault/keyvault.tf

```diff
@@ -47,13 +47,13 @@ resource "azurerm_key_vault" "keyvault" {
     }
   }
 
-  dynamic "contact" {
-    for_each = lookup(var.settings, "contacts", {})
-
-    content {
-      email = contact.value.email
-      name  = try(contact.value.name, null)
-      phone = try(contact.value.phone, null)
-    }
-  }
+  # dynamic "contact" {
+  #   for_each = lookup(var.settings, "contacts", {})
+
+  #   content {
+  #     email = contact.value.email
+  #     name  = try(contact.value.name, null)
+  #     phone = try(contact.value.phone, null)
+  #   }
+  # }
```

Comment:
@MalcolmJMiller  why are we commenting this out?

Reply:
We were having an issue with this code. But it seems to be resolved now. Shouldn't be commented out!